docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -858,7 +890,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Without creating or loading an `Experiment`, it is not possible to run any QCoDeS `Measurement`."
+    "It worths mentioning that what happens if we initilise a database but not create or load an experiment before running a QCoDeS `Measurement`. One of two following outcomes would happen: 1. if the initialised database is empty, then the `Measurement` will throw error because it could not find any `Experiment` in it; 2. if the databse has one or more `Experiment` created in it, then the `Measurement` will be performed and it inherits from the latest `Experiment` name and run number in the database."
```

Comment:
recommendation:
``then the `Measurement` will be performed and it inherits from the latest `Experiment` name and run number in the database.`` --> ``then creating a `Measurement` object will automatically pick up the latest `Experiment` from the database.``

otherwise the reader will be asking "what name?" "what run number if we were talking about the experiment", etc :)